impl/src/test/java/org/ehcache/impl/internal/store/offheap/AbstractOffHeapStoreTest.java

```diff
@@ -439,12 +439,7 @@ public void testComputeExpiresOnAccess() throws StoreAccessException {
       public String apply(String s, String s2) {
         return s2;
       }
-    }, new NullaryFunction<Boolean>() {
-      @Override
-      public Boolean apply() {
-        return false;
-      }
-    });
+    }, () -> false);
```

Comment:
Replace by constant method reference

Reply:
Again, using method references is coming in another PR - otherwise will make this one enormous